source/Calamari.Common/Plumbing/Deployment/PackageRetention/ServerTaskID.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.ComponentModel;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Deployment.PackageRetention;
+
+namespace Calamari.Common.Plumbing.Deployment.PackageRetention
+{
+    //[TypeConverter(typeof(TinyTypeTypeConverter<ServerTaskID>))]
+    public class ServerTaskID : CaseInsensitiveTinyType
```

Comment:
We tend to `Id` instead of `ID`

Reply:
I massively prefer Id over ID, but I could swear I read in our docs somewhere that we should use ID - though apparently I was dreaming, because I can't find it 😅  I'll update these.